src/astro/sql/operators/agnostic_boolean_check.py

```diff
@@ -121,15 +121,15 @@ def prep_boolean_checks_query(
 
         temp_table = (
             select(sqla_checks_object)
-            .select_from(text(table.qualified_name()))
+            .select_from(text(table.qualified_name()))  # type: ignore
```

Comment:
I believe this is an issue with the Dropbox package which defines TypeHints for SQLAlchemy 1.3.24.

I attempted to change `text(table.qualified_name())` by an SQLAlchemy table, and the type checks were happy, but that made the table be represented with double quotes, breaking this feature in the Astro library:
https://github.com/astro-projects/astro/pull/296/commits/f4bd0d1a7d4e0167de025c9cfc9b303bc8baca16#diff-9b06451b9adcb789538c75845a87a78e4cbf63cd055870c659ea8715a18336f4R126